antora-ui-camel/src/partials/catalog.hbs

```diff
@@ -0,0 +1,20 @@
+{{#if navigation.length}}
+  {{#each navigation}}
+    {{#if ../show}}{{#if ./content}}
+      <div class="catalog-entry">
+        <a href="{{~#if (eq ./urlType 'internal')}}{{{relativize ./url}}}{{~else}}{{{./url}}}{{~/if}}">
```

Comment:
Not sure about the purpose of this, we enforce all links to be relative via configuration and a HTML check.

Reply:
That's how Antora handles urls in the nav-tree.hbs partial.

Reply:
With plain Antora UI, you can have links outside the site in the nav-tree, so those can't be relative.  xrefs come through as 'internal'.